src/test/ReactTestUtils.js

```diff
@@ -500,6 +500,7 @@ function makeSimulator(eventType) {
       fakeNativeEvent,
       node
     );
+    event.persist();
```

Comment:
I don't know enough about testing and how this is used, but if it affects behavior for the event received by the callbacks then this seems wrong.